src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -239,7 +239,6 @@ internal void RefreshLibraryView(WebView2 browser)
         /// <param name="text">text to be added to clipboard</param>
         internal void OnCopyToClipboard(string text)
         {
-            dynamoViewModel.Model.ClipBoard.Clear();
```

Comment:
You added this line in https://github.com/DynamoDS/Dynamo/pull/14998  for fixing the problem of copy/paste when using the mouse (click right copy and then click right paste).
if you remove it, will this produce a side-effect behavior on copy/paste using the mouse?